packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -94,25 +117,31 @@
 }
 
 .description {
-  color: var(--calcite-tile-description-text-color);
+  color: var(--calcite-tile-description-text-color, var(--calcite-internal-tile-description-text-color));
   font-weight: var(--calcite-font-weight-regular);
   overflow-wrap: break-word;
 }
 
+calcite-icon {
+  --calcite-icon-color: var(
```

Comment:
the calcite-icon styles can be removed here as it will inherit the default "currentColor" of the component.